mmv1/products/containerattached/Cluster.yaml

```diff
@@ -358,3 +358,17 @@ properties:
             required: true
             description: |
               Namespace of the kubernetes secret containing the proxy config.
+  - !ruby/object:Api::Type::NestedObject
+    name: securityPostureConfig
+    description: |
+      Enable/Disable Security Posture API features for the cluster.
+    allow_empty_object: true
```

Comment:
Does explicitly sending an empty security_posture_config {} block to the API have any specific implications? In general, we'd avoid allowing Terraform users to send empty blocks.

Reply:
> Does explicitly sending an empty security_posture_config {} block to the API have any specific implications?

No, it doesn't. I was following the convention for this resource as other fields also have `allow_empty_object` set.

Reply:
Thanks for confirming! Then I'd suggest that we remove the `allow_empty_object: true` and mark the only subfield `vulnerabilityMode` as required to avoid allowing users to send empty blocks. Thanks!